gui/wxpython/dbmgr/base.py

```diff
@@ -1542,10 +1542,7 @@ def OnDataItemEdit(self, event):
                     column = tlist.columns[columnName[i]]
                     if len(values[i]) > 0:
                         try:
-                            if missingKey is True:
-                                idx = i - 1
-                            else:
-                                idx = i
+                            idx = i - 1 if missingKey is True else i
```

Comment:
```suggestion
                            idx = i - 1 if missingKey else i
```

Reply:
I see in the scope of the function that missingKey is either True or False, so it should be fine with that suggestion